src/main/java/com/jalasoft/jfc/model/strategy/CommandVideoFrameRate.java

```diff
@@ -8,6 +8,8 @@
 
 package com.jalasoft.jfc.model.strategy;
 
+import com.jalasoft.jfc.model.video.VideoCommand;
+
 /**
  * This Class changes the Frame Rate of a video.
```

Comment:
improve this comment This Class changes the Frame Rate of a video to "**This Class manage the Frame Rate of video.**